src/main/java/com/glodblock/github/common/parts/PartFluidTerminal.java

```diff
@@ -19,8 +19,8 @@
 
 public class PartFluidTerminal extends FCBasePart {
     private static final FCPartsTexture FRONT_BRIGHT_ICON = FCPartsTexture.PartFluidTerminal_Bright;
-    private static final FCPartsTexture FRONT_DARK_ICON = FCPartsTexture.PartFluidTerminal_Dark;
-    private static final FCPartsTexture FRONT_COLORED_ICON = FCPartsTexture.PartFluidTerminal_Colored;
+    private static final FCPartsTexture FRONT_DARK_ICON = FCPartsTexture.PartFluidTerminal_Colored;
+    private static final FCPartsTexture FRONT_COLORED_ICON = FCPartsTexture.PartFluidTerminal_Dark;
```

Comment:
May be 
    private static final FCPartsTexture FRONT_DARK_ICON = FCPartsTexture.PartFluidTerminal_Dark;
    private static final FCPartsTexture FRONT_COLORED_ICON = FCPartsTexture.PartFluidTerminal_Colored;
Are you sure you spelled the colors correctly?

Reply:
intended, these names are copied from 1.12 and in 1.7 they are rendered in a different way so i have to swap them